### PR TITLE
Add Windows build

### PR DESCRIPTION
**Pull request by wshanks**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Review by varlackc — COMMENTED** [review]
There are some merge conflicts that will have to be corrected befor approving.
Also, we are missing `the dev_url` and `doc_url`.

**Comment by BastianZim** [comment]
Hmm, I'm not sure if I remember it 100%, but I think it was due to a missing dependency, so it makes sense that it's working now.

Yeah, the tests look great. Thanks for working on that!

The build number would have to be 1, as it will publish all builds again and would overwrite the current ones which might lead to clashes when downloading. I'll change it and merge it.

File: recipe/meta.yaml [hunk 1]
```diff
@@ -12,7 +12,6 @@ source:
 
 build:
   number: 0
```
Comment (BastianZim) [inline recipe/meta.yaml:14]:
```suggestion
  number: 1
```

**Comment by BastianZim** [comment]
@conda-forge-admin, please rerender

**Pull request merged**